### ISSUE TITLE
Spelling error in Get Dash -> OTC section

### ISSUE THREAD

**Issue by Candidculture** [dashpay/dash-website#452]
There appears to be an error on the [Get Dash page](https://www.dash.org/get-dash/). Spillage.t?
<img width="461" alt="screen shot 2018-01-03 at 4 30 31 pm" src="https://user-images.githubusercontent.com/25105397/34544762-ead80474-f0ad-11e7-820a-aa361f3eae9e.png">

**Comment by tungfa** [comment]
hey 
not sure what you mean

Philipp


Philipp Engelhorn
tungfa@dash.org
https://keybase.io/tungfa

**Comment by jhodges10** [comment]
Looks like he means this part

> ... large transactions off the exchanges and free from **slippage.t** things like a new website, pr budget, etc. ...

'slippage.t' is neither a website nor a word in the English language.

That whole paragraph might just need re-writing as it's a bit wordy.

**Comment by tungfa** [comment]
sorry - my bad
got it and will correct that next
ty

**Comment by mrleewong** [comment]
Fixed in #486
